### PR TITLE
Place .htaccess inside of public to avoid manual cp

### PR DESCRIPTION
**Pull request by mdedetrich**
This PR moves the `.htaccess` file into `src/main/public` which allows sbt-paradox to automatically pick up the file as part of the standard web asset pacakging (i.e. `sbt paradox`, paradox inherits this behaviour from sbt-web, see https://github.com/sbt/sbt-web#file-directory-layout)

This allows to get rid of the manual copy step for `.htaccess` (i.e. `cp src/main/paradox/.htaccess content/`) from https://github.com/apache/incubator-pekko-site/wiki/Manual-Publishing .

You can verify this by running `sbt paradox` and seeing the contents of `target/paradox/site/main/*`

### CODE REVIEW

**Comment by pjfanning** [comment]
I tried something similar before and it didn't seem to work but we can try again

**Review by pjfanning — APPROVED** [review]
lgtm

**Comment by mdedetrich** [comment]
> I tried something similar before and it didn't seem to work but we can try again

Probably because you didn't update `excludeFilter` to not exclude hidden files. At least locally I verified that this works.

File: build.sbt [hunk 1]
```diff
@@ -3,3 +3,8 @@ enablePlugins(ParadoxPlugin, PekkoParadoxPlugin)
 resolvers += "Apache Nexus Snapshots".at("https://repository.apache.org/content/repositories/snapshots/")
 
 pekkoParadoxGithub := Some("https://github.com/apache/incubator-pekko-site")
+
+// By default any hidden files placed inside src/main/public are ignored
+inConfig(Assets)(Seq(
+  excludeFilter := excludeFilter.value -- ".htaccess"
```
Comment (pjfanning) [inline build.sbt:9]: I see this is the bit that makes this work

**Pull request merged**